server/test/services/geo/esri/EsriClientTest.java

```diff
@@ -0,0 +1,78 @@
+package services.geo.esri;
+
+import static org.junit.Assert.*;
+import static play.mvc.Results.*;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.collect.ImmutableList;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import java.io.IOException;
+import java.util.*;
+import java.util.concurrent.CompletionStage;
+import org.junit.*;
+import play.libs.Json;
+import play.libs.ws.*;
+import play.routing.RoutingDsl;
+import play.server.Server;
+import services.Address;
+import services.geo.AddressSuggestion;
+import services.geo.AddressSuggestionGroup;
+
+public class EsriClientTest {
+  private EsriClient client;
```

Comment:
**nitpick (non-blocking):** Reverse the order of these to match the order they're set in in the setup. Kinda similar to the requirement for constructor field setting.